backstop/tests/html-tags.html

```diff
@@ -1,110 +0,0 @@
-<!--
```

Comment:
Why'd this get deleted?

Reply:
HTML tags was covering `<a>`, `<abbr>`, and `<img>`.  I moved all of the `<a>` out of that demo into the new anchor demo page.  `<abbr>` wasn't receiving any custom styling and the `@2x`, `@3x` part of the `<img>` was removed a while back, so HTML tags wasn't serving much purpose anymore.